.github/workflows/test.yml

```diff
@@ -71,11 +71,24 @@ jobs:
 
           echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN_PUBLIC_PUBLISH }}" > .npmrc
 
-          yarn version --prerelease --preid ${preid} --allow-branch ${{ github.head_ref }} --no-git-tag-version  --yes --amend
+          yarn version --prerelease --preid ${preid} --allow-branch ${{ github.head_ref }} --no-git-tag-version --yes --amend
+
+          package_version=$(cat package.json \
+            | grep version \
+            | head -1 \
+            | awk -F: '{ print $2 }' \
+            | sed 's/[", ]//g')
+          package=@kong/kongponents@"${package_version}"
+
+          npm show "${package}" >/dev/null 2>&1 && npm_show_status=0 || npm_show_status=1
+          if [ $npm_show_status -eq 0 ]; then
+            echo "Package ${package} is already published. Skipping publishing."
+            exit 0
```

Comment:
We don't want to exit here without returning success as this will also prevent the tests from running.

Reply:
Are you sure? Shouldn’t this just exit this step with a successful status?

Reply:
Yep. 0. Ignore this 🤦🏼‍♂️